backend/app.go

```diff
@@ -82,10 +69,12 @@ const (
 	ServiceSeed  = "molly-wallet-seed"
 	ServicePKey  = "molly-wallet-pkey"
 
-	MainnetBlockExplorerURL = "https://block-explorer.constellationnetwork.io"
-	TestnetBlockExplorerURL = "https://api-be.exchanges.constellationnetwork.io"
-	MainnetLoadBalancerURL  = "http://lb.constellationnetwork.io:9000"
-	TestnetLoadBalancerURL  = "http://lb.exchanges.constellationnetwork.io:9000"
+	//TODO MainnetBlockExplorerURL = "https://be.constellationnetwork.io"
+	MainnetBlockExplorerURL = "https://be-testnet.constellationnetwork.io"
+	TestnetBlockExplorerURL = "https://be-testnet.constellationnetwork.io"
+	//TODO MainnetLoadBalancerURL  = "http://lb.constellationnetwork.io:9000"
+	MainnetLoadBalancerURL = "http://lb-testnet.constellationnetwork.io:9010"
```

Comment:
```suggestion
	MainnetLoadBalancerURL = "http://lb-mainnet.constellationnetwork.io:9010"
```

Reply:
@AlexBrandes - this is on purpose. This will be the last fix.
I left a TODO there...